apps/studio/components/ui/FilterPopover.tsx

```diff
@@ -38,9 +38,10 @@ export const FilterPopover = ({
       header={<div className="prose text-xs">Select {name.toLowerCase()}</div>}
       overlay={
         <>
-          <div className="space-y-4 px-3 py-3 min-w-[170px]">
+          <div className="space-y-4 min-w-[170px]">
```

Comment:
The outline rings for the checkboxes were cut off.

![Screenshot 2024-04-09 at 15 36 26](https://github.com/supabase/supabase/assets/568291/feb38d05-7784-482b-b7db-ca62cdedbaed)